src/Particles.jl

```diff
@@ -95,20 +99,20 @@ function read_particle_csv(filepath::AbstractString)
         name = row[16]
         quarks = row[17]
         latex = row[18]
-        dct_particles[pdgid] = ParticleInfo(pdgid,
-                                            mass, 
-                                            width, 
-                                            charge,
-                                            isospin,
-                                            parity,
-                                            gparity,
-                                            cparity,
-                                            antiprop,
-                                            rank,
-                                            status,
-                                            name,
-                                            quarks,
-                                            latex)
+        dct_particles[pdgid] = Particle(pdgid,
```

Comment:
When introducing the `ParticleInfo` type I avoided to name it `Particle`  as this could be used for individual particles (e.g. including a position and momentum)

Reply:
Yes, but I exactly explit this ;)